articles/cognitive-services/Computer-vision/QuickStarts/python-print-text.md

```diff
@@ -94,12 +94,28 @@ for word in word_infos:
     bbox = [int(num) for num in word["boundingBox"].split(",")]
     text = word["text"]
     origin = (bbox[0], bbox[1])
-    patch  = Rectangle(origin, bbox[2], bbox[3], fill=False, linewidth=2, color='y')
+    patch = Rectangle(origin, bbox[2], bbox[3],
+                      fill=False, linewidth=2, color='y')
     ax.axes.add_patch(patch)
     plt.text(origin[0], origin[1], text, fontsize=20, weight="bold", va="top")
 plt.axis("off")
 ```
 
+## Upload image from local storage
+
+In order to analyse a local image you need to set the Content-Type to octet-stream and instead of json put a byte array inside the post request.
```

Comment:
rewrite to:

If you want to analyze a local image, set the Content-Type header to `application/octet-stream`, and set the request body to a byte array instead of JSON data.

And then we're good to go. Thanks for fixing the other stuff

Reply:
Done! 
Again thank you for your kind assistance. It took me a while to understand what the issue was.